src/tests/refinement/helper.rs

```diff
@@ -61,6 +62,26 @@ pub fn json_run_query(path: &str, query: &str) -> Result<QueryResult, Executable
         query: Option::from(query),
         comment: "".to_string(),
     };
+    // After implementing clock reduction on component level, a few tests are failing due to
+    // inconsistencies with initial state and global clock. Turn boolean true to ignore inconsistencies
```

Comment:
It might be difficult for some to parse what boolean should be true based on this sentence.
Maybe it should use the name of the boolean.
Maybe it should also have a FIXME tag or something like that to draw attention to this problem, i could imagine some confusion for new developers